ingestion/functions/template.yaml

```diff
@@ -25,7 +25,7 @@ Resources:
         - AWSLambdaFullAccess
         - S3WritePolicy:
             BucketName: !Ref RawSourcesBucket
-      Timeout: 60 # Seconds
+      Timeout: 600 # Seconds
```

Comment:
Probably better just to remove this. I had it at 60 originally for development purposes, but in prod we probably want it to take as much time as it needs (up to the 900 second limit).

Reply:
ok fair enough, let's remove it.